automl/automl-script/readme.md

```diff
@@ -21,24 +21,28 @@ be automatically done:
   lower than 50 (or 30 if `shallow_search` is `True`)
 -  `Model Selection` Using OptiML to find the best models and using
   the top 3 models to create a `Fusion` model to predict all the test
-  dataset instances. If a validation dataset is given, this script
-  will return an evaluation of the final model too.
+  dataset instances. The script will return an evaluation of the final
+  model too.
```

Comment:
I would replace the last line of the previous paragraph and place a new item: 

- Fusion evaluation: Evaluating the Fusion using the data in the holdout dataset
- Fusion prediction: Scoring the input data provided in the test dataset (if set) using the final Fusion model.